Farfetch/garment
# Include Typescript sources in packages to support local debugging

# Description

Includes the src directory in all packages which also included lib.

No unit tests added, as this is a functionality of the package installer itself.

Fixes #10

## How Has This Been Tested?

The following steps were done for both `@garment/cli` and `@garment/plugin-runner-ts`:

1. Use yarn pack in package directory
2. Install zip files resulting from yarn pack in a repo that uses garment.  Verify that src files are included in correct location upon installation.
3. Debug original TypeScript files in Chrome Node inspector.

## Checklist:


### Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)
